Odysee/Utils/Extensions.swift

```diff
@@ -107,3 +107,76 @@ extension Optional {
         return self ?? defaultValue()
     }
 }
+
+// These extensions are useful for Lbry.swift – so that we can support old methods that haven't
+// migrated to their own Params type yet and still use dictionaries.
+extension NSString: Encodable {
+    public func encode(to encoder: Encoder) throws {
+        try (self as String).encode(to: encoder)
+    }
+}
+
+extension NSNull: Encodable {
+    public func encode(to encoder: Encoder) throws {
+        var c = encoder.singleValueContainer()
+        try c.encodeNil()
+    }
+}
+
+extension NSDictionary: Encodable {
+    public func encode(to encoder: Encoder) throws {
+        var c = encoder.container(keyedBy: CodingKeys.self)
+        for (k, v) in self {
+            guard let k = k as? String else {
+                throw EncodingError.invalidValue(k, .init(codingPath: encoder.codingPath,
+                                                          debugDescription: "NSDictionary key"))
+            }
+            let nestedEncoder = c.superEncoder(forKey: .init(stringValue: k))
+            guard let v = v as? Encodable else {
+                throw EncodingError.invalidValue(v, .init(codingPath: nestedEncoder.codingPath,
```

Comment:
/Users/akinwale/Dev/iOS/Odysee/Odysee/Utils/Extensions.swift:136:50: Variable declared in 'guard' condition is not usable in its body